server.js

```diff
@@ -1,9 +1,20 @@
+require('dotenv').config();
 const browserSync = require('browser-sync');
 const config = require('./app/config');
 const { App } = require('./app');
 const { AuthProvider } = require('./app/authProvider');
 const { routes } = require('./app/routes');
 
+Object.keys(config).map((configKey) => {
+  if (config[configKey]) {
+    // eslint-disable-next-line no-console
+    console.log(`${configKey} set to ${config[configKey]}`);
```

Comment:
use logger?

Reply:
also tried that 😄 but kept saying logger.info is not a function or something